content/en/docs/concepts/storage/volume-snapshot-classes.md

```diff
@@ -39,6 +39,10 @@ request a particular class. Administrators set the name and other parameters
 of a class when first creating VolumeSnapshotClass objects, and the objects cannot
 be updated once they are created.
 
+{{< note >}}
+Installation of the CRDs are the responsibility of the Kubernetes distribution. Without the required CRDs present the `VolumeSnapshotClass` fails. 
```

Comment:
It's not clear to me what "responsibility of the Kubernetes distribution" means here? Do you mean responsibility of the cluster administrator?

```suggestion
Installation of the CRDs is the responsibility of the Kubernetes distribution. Without the required CRDs present, the `VolumeSnapshotClass` fails. 
```

Reply:
> It's not clear to me what "responsibility of the Kubernetes distribution" means here? Do you mean responsibility of the cluster administrator?

kubernetes distribution is a collection of kubernetes and related technologies and services that we need to use together for deploy and use containers.

Reply:
"the `VolumeSnapshotClass` fails" -> the creation of a `VolumeSnapshotClass` fails

Reply:
Change the note as suggested.Thanks